guidelines/sc/22/focus-appearance-minimum.html

```diff
@@ -5,15 +5,14 @@ <h4>Focus Appearance (Minimum)</h4>
     <p class="conformance-level">AA</p>
 	<p class="change">New</p>
     
-    <p>When <a>user interface components</a> receive keyboard focus, an area of the focus indicator meets the following:</p>
+    <p>When <a>user interface components</a> receive keyboard focus, the focus indicator meets the following:</p>
     <ul>
-        <li><strong>Minimum area:</strong> The area is either:
+        <li><strong>Contrasting area:</strong> The focus indicator has an area with a contrast ratio of at least 3:1 between the colors when the component is focused and it is not focused, and that area is at least as large as either:
```

Comment:
> when the component is focused and it is not focused

maybe me, but i'd add an extra "when" there

> when the component is focused and when it is not focused

Reply:
> and that area

perhaps better

> and its area

Reply:
+1/-1 means I can go either way, but I prefer "that" here

Reply:
"that area" to me implies that you're defining the "focus indicator" as an area.

"its area" treats "focus indicator" as more than just an area, and the area as one of its properties. if that makes sense...

Reply:
"the area"?

Reply:
"the area of the indicator"

Reply:
That wouldn't work, the area in scope can be smaller than the entire focus indicator (e.g. with a gradient fade-out). I think maybe it's best as 'that area', because it scopes that at the start of the sentence.